### DISCUSSION TITLE
Chromium profiles filling up hard disk

### DISCUSSION THREAD

**Discussion by dolbex** [spatie/browsershot#616] [Q&A]
Quick question to see if anyone has run into this. We recently installed browsershot and it's working fantastic. However, it seems like chrome/puppeteer are creating a needless amount of chrome profiles in our `/tmp/snap.chromium/tmp` directory (like 8 gigs). Anyone experienced this? Maybe we misconfigured something?

**Comment by haprodev** [comment]
I have also encountered this problem.
Can you help us?
